### PR TITLE
[python] Update to esbuild script

### PR DESCRIPTION
**Pull request by cb1kenobi**

### CODE REVIEW

File: packages/python/build.mjs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { esbuild } from '../../utils/build.mjs';
+
+await esbuild({ bundle: true, external: ['@vercel/build-utils'] });
```
Comment (styfle) [inline packages/python/build.mjs]: Don't you need to mark all the `dependencies` in package.json as external too?

File: .changeset/slow-tips-jam.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
Comment (styfle) [inline .changeset/slow-tips-jam.md:3]: This should be patch so that it gets published

Comment (cb1kenobi) [reply .changeset/slow-tips-jam.md:3]: Fixed

File: .changeset/slow-tips-jam.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+'@vercel/go': patch
```
Comment (styfle) [inline .changeset/slow-tips-jam.md]:
```suggestion
'@vercel/python': patch
```

**Review by styfle — APPROVED** [review]

File: packages/python/build.mjs (hunk 1 above)
Comment (cb1kenobi) [reply packages/python/build.mjs]: Non-issue... using the new builder-build.mjs script

**Pull request merged**